app/views/includes/omic-organisation-overview.njk

```diff
@@ -4,7 +4,7 @@
                 <th class="govuk-table__header sln-table-org-level" aria-sort="none" id="name">{{ childOrganisationLevelDisplayText }}</th>
                 <th class="govuk-table__header govuk-table__header--numeric" aria-sort="descending" id="cases">OMIC Cases</th>
                 <th class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="custody">Total Points while in Custody</th>
-                <th class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="licence">Anticipated Points while on Licence</th>
+                <th class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="license">Anticipated Points while on Licence</th>
```

Comment:
Isn't this the wrong way around?

Reply:
changed the id's by mistake in this file